pkgs/top-level/all-packages.nix

```diff
@@ -1140,6 +1140,8 @@ with pkgs;
 
   cfdyndns = callPackage ../applications/networking/dyndns/cfdyndns { };
 
+  cht-sh = callPackage ../tools/misc/cht.sh { };
```

Comment:
directory name: cht.sh
package name: cht.sh
attribute name: cht-sh

is that ok?

Reply:
I'd say yes. `cht.sh` is interpreted as `cht = { sh = <your package>; };` and this can be fairly confusing IMHO

Reply:
Updated the commit message